taxi/forms.py

```diff
@@ -1,52 +1,64 @@
 from django import forms
 from django.contrib.auth import get_user_model
 from django.contrib.auth.forms import UserCreationForm
-from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
 
-from taxi.models import Car, Driver
+from taxi.models import Driver, Car
 
 
-class CarForm(forms.ModelForm):
-    drivers = forms.ModelMultipleChoiceField(
-        queryset=get_user_model().objects.all(),
-        widget=forms.CheckboxSelectMultiple,
-    )
-
-    class Meta:
-        model = Car
-        fields = "__all__"
+DRIVER_LICENSE_VALIDATOR = RegexValidator(
+    regex=r"^[A-Z]{3}[\d]{5}$",
+    message="Enter a valid value license number:"
+            " 1.Consist only of 8 characters"
+            " 2.First 3 characters are uppercase letters"
+            " 3.Last 5 characters are digits", )
```

Comment:
The message for `DRIVER_LICENSE_VALIDATOR` could be more concise. Consider rephrasing it to something like: 'License number must be 8 characters long, with the first 3 being uppercase letters and the last 5 digits.'